internal/config/config_test.go

```diff
@@ -788,41 +799,266 @@ func TestParse(t *testing.T) {
 			},
 		},
 		{
-			desc: "bad community literal (wrong format)",
+			desc: "bad community literal (wrong format) - in BGP adv",
 			crs: ClusterResources{
+				Pools: []v1beta1.IPPool{
```

Comment:
How about having a variable with this pool and using it in all these tests?
The volume of the test will be smaller, and the content is not really relevant for the test.

Reply:
Done.